docs/source/reference/core/debug.rst

```diff
@@ -6,20 +6,40 @@ Debug mode
 In debug mode, Chainer checks values of variables on runtime and shows more
 detailed error messages.
 It helps you to debug your programs.
-Instead it requires additional overhead time.
+However, it requires some additional overhead time.
 
-In debug mode, Chainer checks all results of forward and backward computation, and if it founds a NaN value, it raises :class:`RuntimeError`.
+You can enable debug mode with :func:`chainer.using_config`:
+
+.. code-block:: python
```

Comment:
How about making snippet code testable?

Reply:
The code is very short and the content is only ```...```. Isn't it enough?

Reply:
It is because I want to check if all code snippets are correct automatically especially when the interface is changed.

Reply:
How can I write?
If extra `>>>` and `...` are printed, isn't it confusing?
```
>>> with chainer.using_config('debug', True):
...     ...
```

Reply:
I found I can just replace it with `testcode`. `>>>` is not needed and still tested.